website/docs/r/windows_web_app_slot.html.markdown

```diff
@@ -754,7 +754,7 @@ A `site_config` block supports the following:
 
 * `scm_use_main_ip_restriction` - (Optional) Should the Windows Web App Slot `ip_restriction` configuration be used for the SCM also.
 
-* `use_32_bit_worker` - (Optional) Should the Windows Web App Slotuse a 32-bit worker. Defaults to `true`.
+* `use_32_bit_worker` - (Optional) Should the Windows Web App Slot use a 32-bit worker. The default value varies from different service plans.
```

Comment:
We're defaulting this value in the file above on line 244, so whilst _the service_ might have a different default value for this field - Terraform has a singular fixed value here?